src/test/java/org/jenkinsci/plugins/rundeck/RundeckNotifierTest.java

```diff
@@ -341,6 +356,12 @@ private SubversionSCM createScm() throws Exception {
 
     private void addScmCommit(FilePath workspace, String commitMessage) throws Exception {
         SVNClientManager svnm = SubversionSCM.createSvnClientManager((ISVNAuthenticationProvider)null);
+        SVNAuthenticationManager mgr = new SVNAuthenticationManager(new File(jenkins.root, ".svn"), "user", "password");
```

Comment:
Related?

Reply:
yeah. Tests fail with NPE otherwise. Seems to be an old change in SVNKit